coroutines-extra/src/test/scala/AsyncAwaitTests.scala

```diff
@@ -0,0 +1,137 @@
+package org.coroutines.extra
+
+
+
+import org.coroutines._
+import org.scalatest._
+import scala.concurrent.{ Await, Future }
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+
+
+class AsyncAwaitTests extends FunSuite with Matchers {
+  class TestException extends Throwable
+
+  /** Source: https://git.io/vorXv
+   *  The use of Async/Await as opposed to pure futures allows this control flow
+   *  to be written more easily.
+   *  The execution blocks when awaiting for the result of `f1`. `f2` only blocks
+   *  after `AsyncAwait.await(f1)` evaluates to `true`.
+   */
+  test("simple test") {
+    val future = AsyncAwait.async {
```

Comment:
Can you do `import AsyncAwait._` at the beginning of the `AsyncAwaitTests` class?